src/components/Card/tests/Card.test.tsx

```diff
@@ -6,8 +6,10 @@ import {contentContextTypes} from '../../../types';
 
 describe('<Card />', () => {
   it('has a child with contentContext', () => {
-    const Child: React.SFC<{}> = (_props, context) =>
-      context.withinContentContainer ? <div /> : null;
+    const Child: React.SFC = (_props, context) => {
```

Comment:
ping @cartogram This sort of thing feels like a false positive on `shopify/jest/no-if`. would you be able to look into it?

Reply:
Yeah, it is. Can you copy and paste this example into an issue on the eslint-plugin-shopify?

For now, keep the inline disable and when I fix the rule, the disable comment should be flagged but the no-disable-comments rule and you can remove it then. Thanks Ben!

Reply:
Done! https://github.com/Shopify/eslint-plugin-shopify/issues/311